src/core/qgsproject.cpp

```diff
@@ -1082,6 +1082,9 @@ bool QgsProject::read()
     mTranslator.reset( nullptr );
   }
 
+  // Set home path
+  setPresetHomePath( homePath() );
```

Comment:
Is there a risk that this might set an absolute path on projects?
I'm not completely sure, where the homepath would be set before in the read() process, but maybe it's safer to do something like this?

```
QgsProjects::read()
{
   const QString oldHomePath = homePath()
   ... read ...
   if ( oldHomePath != homePath() )
     emit homePathChanged();
```